src/modules/location.rs

```diff
@@ -4,6 +4,35 @@ use reqwest::Client;
 use serde::Deserialize;
 
 use super::{config::Config, localization::Locales};
+use crate::modules::api::Api;
```

Comment:
I imported my newly introduced trait and created some structs implementing it.  I prefer the `crate::` notation but I can also change it as required.  At the moment, this is a nice indicator for the changes I submitted.

Reply:
Sympathizing with this as it was also my preferred style and I used to do that most of the time until I gave in because my rust tools are usually went with super for auto imports. I probably should have checked if I can change the default behavior to keep a uniform style before giving in.